impl/src/main/java/org/jboss/weld/interceptor/proxy/InterceptorMethodHandler.java

```diff
@@ -27,13 +27,15 @@
 public class InterceptorMethodHandler implements StackAwareMethodHandler, Serializable {
 
     private static final long serialVersionUID = 1L;
+    public static final String INTERCEPTOR_BINDINGS_KEY = "org.jboss.weld.interceptor.bindings";
```

Comment:
This could be moved to API (if we do the API change for 2.4). In any case we should document this.

Reply:
https://issues.jboss.org/browse/WELD-2436